config/routes.rb

```diff
@@ -1,21 +1,10 @@
 Rails.application.routes.draw do
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
-
-  # You can have the root of your site routed with "root"
```

Comment:
@FrankCardillo sounds good!

In the future, I would recommend reviewing your own pull requests through GitHub.  This way you can see **exactly** what you're changing, and it will be easier to catch little things like this. 👍